framework/platform/bpl/include/bpl/bpl_cfg.h

```diff
@@ -91,6 +91,7 @@ namespace bpl {
 #define DEFAULT_BAND_STEERING 1
 #define DEFAULT_DFS_REENTRY 1
 #define DEFAULT_CLIENT_ROAMING 1
+#define DEFAULT_PERSISTENT_DB 0
```

Comment:
Since you mentioned in another comment:
```
the original file convention is c based interface - and we no longer force the interface to be c and can now provide a c++ style interface to any new APIs
```
Use `constexpr` instead of defines. Based on your comment you can't say "this is the convention in this file" :)

Reply:
replaced all defines with `constexpr` - thanks